### PR TITLE
Update README and tweak OEP language

### PR DESCRIPTION
**Pull request by sarina**
Updates the README file to comply with OEP-55: https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055/decisions/0003-readme-specification.html. I also noticed that the language defining OEPs casts them as purely technical decisions. However, with OEPs such as Core Contributors and Core Product, I think we've found that OEPs are transcending pure technical decisions. I've added the language of 'community decisions' to address this. @feanil and anyone else welcome.

Review open until Mon 13 Feb

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -39,3 +73,50 @@ documents in a hidden toctree near where they are referenced::
...
+
+.. _Open edX Code of Conduct: https://openedx.org/code-of-conduct/
+
+People
+******
+
+See the `open-edx-proposals Backstage page`_ for more information about who the
+maintainers of this repo are and how to get in touch with them.
+
+.. _open-edx-proposals Backstage page: https://backstage.openedx.org/catalog?filters%5Bkind%5D=component&filters%5Buser%5D=all
```
Comment (sarina) [inline README.rst:104]: Link will be corrected once https://github.com/openedx/open-edx-proposals/pull/441 merges

File: README.rst [hunk 2]
```diff
@@ -39,3 +73,50 @@ documents in a hidden toctree near where they are referenced::
...
+
+Reporting Security Issues
+*************************
+
+Please do not report security issues in public. Please email security@tcril.org.
+
+
+.. |license-badge| image:: https://img.shields.io/badge/License-CC_BY--SA_4.0-lightgrey.svg
```
Comment (sarina) [inline README.rst:112]: The recommended link,  https://img.shields.io/github/license/openedx/open-edx-proposals.svg, didn't work. It didn't recognize the Creative Commons license.

File: README.rst [hunk 3]
```diff
@@ -1,21 +1,55 @@
...
 
 .. _Open edX Proposals (OEPs): https://open-edx-proposals.readthedocs.io
 
-Testing locally
+Getting Started
 ***************
 
+This repository holds a bunch of text files, each of which represent an OEP or
+supplementary documentation. For browsing these OEPs, we recommend you read them
+on the website.
```
Comment (feanil) [inline README.rst:23]:
```suggestion
in `their published form <https://open-edx-proposals.readthedocs.io>`_ on ReadTheDocs.
```

**Review by feanil — APPROVED** [review]
One suggestion to take or leave, otherwise looks good to merge.

**Pull request merged**